0019/README.md

```diff
@@ -152,6 +154,133 @@ Endpoint metadata get response message payload MUST be an Avro-encoded object wi
 }
 ```
 
-### Timeout and retry
+
+## Partial endpoint metadata update
+
+Completing a partial metadata update request will update or create only the endpoint metadata key-value pairs present in the request payload using [JSON Patch](https://tools.ietf.org/html/rfc6902). Metadata keys must be non-empty strings containing Latin letters, digits, and underscores only.
```

Comment:
Remove this paragraph since:
1. "Completing a partial metadata update request will update or create..." sounds not like RFC but the implementation description.
2. Requirements to metadata keys are already defined in 10/EPMP, there is no need to duplicate it here.

Reply:
Partial endpoint metadata update request MUST partially update endpoint metadata using the JSON Patch defined in [RFC 6902](https://tools.ietf.org/html/rfc6902).